include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -398,6 +419,51 @@ namespace grl { namespace robot { namespace arm {
       volatile std::size_t m_attemptedCommunicationConsecutiveSuccessCount = 0;
 
 
+
+      void setPositionControlMode()
```

Comment:
comments from KukaDriver.hpp applies in the same way to the new functions here too